test/unit/remote_control_test.lua

```diff
@@ -530,7 +530,6 @@ function g.test_call()
         )
         t.assert_not(ok)
         t.assert_equals(type(err), 'cdata')
-        t.assert_equals(err.code, box.error.PROC_LUA)
```

Comment:
This way you just remove a part of tests. IMO it's not a solution. I think it's appropriate to support test cases for all versions based on _TARANTOOL value.

Personally, I'm not sure we should avoid tarantool internals consideration. Especially in case when implement Tarantool-compatible interfaces.